### ISSUE TITLE
First pilot selection title is not clear, and actions are hidden

### ISSUE THREAD

**Issue by ohadch** [ohadch/gliding-action-page-v3#155]
![image](https://github.com/user-attachments/assets/b68b8940-5f1a-49bb-bcdc-403d34262bb0)

**Comment by ohadch** [comment]
Fixed by adding the following to the `Autocomplete` component:

```
ListboxProps={{
                                  style: {
                                      maxHeight: 300,
                                  }
                              }}
```

<img width="824" alt="image" src="https://github.com/user-attachments/assets/b80fa0b6-ab8c-46fd-b77d-7243604da0b5">

**Comment by ohadch** [comment]
Also adds a title for the input for better clarity:

<img width="848" alt="image" src="https://github.com/user-attachments/assets/28e4466d-10f9-4031-a5ca-6cc8a4bb1140">

**Comment by ohadch** [comment]
Also beautifies the following dialogs:

Tow airplane edit dialog:
<img width="544" alt="image" src="https://github.com/user-attachments/assets/d2cfd10d-4699-477c-9706-f3c45d214de9">

Payment settlement dialog:
<img width="1004" alt="image" src="https://github.com/user-attachments/assets/a69199c0-0958-4a29-b59f-c1af8f302deb">

**Comment by ohadch** [comment]
Emphasizes the payment settlement alert:
<img width="1550" alt="image" src="https://github.com/user-attachments/assets/fae57fa3-fbeb-4fd6-84e2-547ecc2b3643">

**Comment by ohadch** [comment]
Tidies up the flight dispatch dialog:

<img width="1135" alt="image" src="https://github.com/user-attachments/assets/5f4a82d7-1b70-45da-b697-a520db6938ad">

**Comment by ohadch** [comment]
Emphasizes the glider's call sign:
<img width="1549" alt="image" src="https://github.com/user-attachments/assets/ea861fc9-794f-42bf-ad6b-756b406e079e">

**Comment by ohadch** [comment]
closed by #195
